src/m365/spo/commands/list/list-view-list.spec.ts

```diff
@@ -8,19 +8,28 @@ import { Logger } from '../../../../cli/Logger';
 import Command, { CommandError } from '../../../../Command';
 import request from '../../../../request';
 import { pid } from '../../../../utils/pid';
+import { urlUtil } from '../../../../utils/urlUtil';
+import { formatting } from '../../../../utils/formatting';
 import { sinonUtil } from '../../../../utils/sinonUtil';
 import commands from '../../commands';
 const command: Command = require('./list-view-list');
 
 describe(commands.LIST_VIEW_LIST, () => {
+  const listId = '1f187321-f086-4d3d-8523-517e94cc9df9';
+  const webUrl = 'https://contoso.sharepoint.com/sites/ninja';
+  const listTitle = 'Documents';
+  const listUrl = '/sites/ninja/Shared Documents';
+  const listViewOutput = {
```

Comment:
Let's rename this variable to `listViewResponse` and move it between `#region Mocked Responses` and `#endregion` (like with app-set.spec.ts)